wolfhsm/wh_client.h

```diff
@@ -617,6 +617,82 @@ int wh_Client_AesCmacVerify(Cmac* cmac, const byte* check, word32 checkSz,
 int wh_Client_SetKeyCmac(Cmac* key, whNvmId keyId);
 #endif
 
+/* Counter functions */
+int wh_Client_CounterInitRequest(whClientContext* c, whNvmId keyId,
+    uint32_t counter);
+int wh_Client_CounterInitResponse(whClientContext* c, uint32_t* counter);
+/**
+ * @brief Creates and initializes a counter with the value set in counter.
+ *
+ * This function creates/resets a counter with the supplied keyId and gives it
+ * the value stored in counter at the start of the call.
+ *
+ * @param[in] c Pointer to the whClientContext structure.
+ * @param[in] keyId Key ID to be associated with the CMAC key.
```

Comment:
Copy pasta with the CMAC.  Consider naming the parameter CounterId instead of KeyId.